deep_daze/deep_daze.py

```diff
@@ -436,11 +401,14 @@ def save_image(self, epoch, iteration, img=None):
         sequence_number = self.get_img_sequence_number(epoch, iteration)
 
         if img is None:
-            #img = normalize_image(self.model(self.clip_encoding, return_loss=False).cpu().float()).clamp(0., 1.)
             img = self.model(self.clip_encoding, return_loss=False).cpu().float().clamp(0., 1.)
         self.filename = self.image_output_path(sequence_number=sequence_number)
-        save_image(img, self.filename)
-        save_image(img, f"{self.textpath}.png")
+        
+        pil_img = T.ToPILImage()(img.squeeze())
```

Comment:
I'm assuming this change is helpful for the video creation you're ultimately after? I know some folks are sticklers about compression though. Particularly when you want to do post-processing. I'm personally fine with it however.

Reply:
You're right, this choice is debatable. The image now saves faster (negligible) and the files-size is smaller. The smaller file-size is beneficial for me, as I am generating a lot of images. Maybe we could just solve this by allowing another option for the user to choose between PNG and JPG?

Reply:
For sure, that works.